pkg/internal/apis/acme/types_issuer.go

```diff
@@ -46,6 +51,21 @@ type ACMEIssuer struct {
 	Solvers []ACMEChallengeSolver `json:"solvers,omitempty"`
 }
 
+// ACMEExternalAcccountBinding is a reference to a CA external account of the ACME
+// server.
+type ACMEExternalAccountBinding struct {
+	// Key ID is the ID of the CA key that the external account is bound to.
+	KeyID string `json:"keyID,omitempty"`
+
+	// Key is a base64 encoded symmetric key that authenticates the account with
+	// an external account.
+	Key string `json:"key,omitempty"`
```

Comment:
This feels like a sensitive value, given it is used to authenticate/bind a registration to some external system?

Reply:
Also, how come this is a base64 encoded string? Does this really mean it is a `[]byte`?

Reply:
Indeed, makes sense to move to a secret

Reply:
In theory you can should be able to use raw []byte data for your symmetric MAC key. base64 encoding makes then becomes a requirement